### PR TITLE
Dual axes tutorial

### PR DESCRIPTION
**Pull request by natashadecoste**
### Updates
- list
- out
- updates
- here (and don't forget to link the issues)

### Review checklist (for reviewers only)
- [ ] Demos all features
- [ ] Documented/annotated
- [ ] Matches UI/UX specs
- [ ] Meets the [code style guide](https://github.com/carbon-design-system/carbon-charts/wiki/Code-style-guide)
- [ ] Accessible
- [ ] Mobile first (responsive)
- [ ] RTL support (bidirectional text)
- [ ] Performant (limited bloat)

### CODE REVIEW

File: packages/core/stories/tutorials/combo-charts.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import marked from "marked";
+
+export const comboChartTutorial = {
+	name: "Combo Chart Tutorial",
+	content: marked(`
+# Combo charts
+Combo charts allow users to visually compare complex datasets by superimposing different graphing components within one chart.
+Combination charts **may or may not** use dual axes although it is common that they do. For more information on dual axes with **carbon-charts**, please see this tutorial ____.
```
Comment (theiliad) [inline packages/core/stories/tutorials/combo-charts.ts:8]: `____`?

File: packages/core/stories/tutorials/combo-charts.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+];
+
+const comboOptions = {
+	...otherOptions // global options apply to all the charts listed in chartTypes (where relevant)
+	axes: {...}
+	chartTypes: [ // takes an array of objects
+		{
+			type: "area" // use a valid ChartType from enums
+			options: {}, // component specific options (i.e, turn on the scatter points, adjust scatter radius)
+			datasets: ["Dataset 1"] // the datasets that we want to map with this type of chart
```
Comment (theiliad) [inline packages/core/stories/tutorials/combo-charts.ts:48]: corresponding

File: packages/core/stories/tutorials/dual-axes.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	content: marked(`
+# Dual Axes
+If a chart is considered to be a **dual axes** chart, it means that the chart uses 2 **different** axes for visually mapping out ranges for the data.
+Careful consideration should be taken when using Dual Axes charts because they can more often misrepresent the data.
+
+Please refer to dataviz guidelines here for more information on IBM's dataviz guidelines.
+
+
+
+___
```
Comment (theiliad) [inline packages/core/stories/tutorials/dual-axes.ts:14]: ?

**Review by theiliad — CHANGES_REQUESTED** [review]

**Review by theiliad — DISMISSED** [review]

**Review by theiliad — APPROVED** [review]

File: packages/core/stories/tutorials/dual-axes.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+____
+
+## Setting up a dual axes chart
+
+When using multiple Y or X axes, it is best to specify which axes will be primary (used for ranges or domain).
+Below we define options for an axis chart with **two dual axes (both vertical)** and one horizontal axis (bottom). In this case, we want to
+set up the horizontal axis (bottom) as the domain, and have the datasets' ranges mapped to the different vertical axes - "Temperature" on the left axis and "Products Sold" on the right.
+
+Use the \`main\` option to define the \`left\` axis as the primary axis. The axis defined as \`main\` will be used to map **all datasets** until we set the
+\`correspondingDatasets\` on the non-primary (secondary) axis. If there is no \`correspondingDatasets\` set on the secondary axis, then the chart will render
```
Comment (zvonimirfras) [inline packages/core/stories/tutorials/dual-axes.ts:22]:
```suggestion
\`correspondingDatasets\` on the non-primary (secondary) axis. If \`correspondingDatasets\` isn't set on the secondary axis, then the chart will render
```

File: packages/core/stories/tutorials/dual-axes.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+		  ]
+		}
+	  },
+};
+\`\`\`
+
+Note: The library tries to determine the orientation/primary axes **without the use of \`main\`**,
+however it is best to be explicit when working with dual axes charts. If there is no \`main\` specified, the library
+will try to determine the best primary/secondary assignments to make which could result in some confusion down the road.
+
```
Comment (zvonimirfras) [inline packages/core/stories/tutorials/dual-axes.ts:53]:
```suggestion
```

File: packages/core/stories/tutorials/dual-axes.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+Please refer to dataviz guidelines [here](https://www.carbondesignsystem.com/data-visualization/getting-started) for more information on IBM's best practices.
+
+____
+
+## Setting up a dual axes chart
+
+When using multiple Y or X axes, it is best to specify which axes will be primary (used for ranges or domain).
+Below we define options for an axis chart with **two dual axes (both vertical)** and one horizontal axis (bottom). In this case, we want to
+set up the horizontal axis (bottom) as the domain, and have the datasets' ranges mapped to the different vertical axes - "Temperature" on the left axis and "Products Sold" on the right.
```
Comment (zvonimirfras) [inline packages/core/stories/tutorials/dual-axes.ts:19]:
```suggestion
set up the horizontal axis (bottom) as the domain, and have the datasets' ranges mapped to the different vertical axes - "Temperature" on the left axis and "Products sold" on the right.
```

File: packages/core/stories/tutorials/dual-axes.ts [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+		  mapsTo: temperature,
+		  scaleType: linear,
+		  main: true // this will be the primary axis
+		},
+		right: {
+		  mapsTo: value,
+		  scaleType: linear,
+		  // the dataset "Products Sold" will map to this axis instead
+		  correspondingDatasets: [
+			"Products Sold"
```
Comment (zvonimirfras) [inline packages/core/stories/tutorials/dual-axes.ts:43]:
```suggestion
			"Products sold"
```

File: packages/core/stories/tutorials/dual-axes.ts [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+		},
+		left: {
+		  mapsTo: temperature,
+		  scaleType: linear,
+		  main: true // this will be the primary axis
+		},
+		right: {
+		  mapsTo: value,
+		  scaleType: linear,
+		  // the dataset "Products Sold" will map to this axis instead
```
Comment (zvonimirfras) [inline packages/core/stories/tutorials/dual-axes.ts:41]:
```suggestion
		  // the dataset "Products sold" will map to this axis instead
```

File: packages/core/stories/tutorials/dual-axes.ts [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+		  correspondingDatasets: [
+			"Products Sold"
+		  ]
+		}
+	  },
+};
+\`\`\`
+
+Note: The library tries to determine the orientation/primary axes **without the use of \`main\`**,
+however it is best to be explicit when working with dual axes charts. If there is no \`main\` specified, the library
```
Comment (zvonimirfras) [inline packages/core/stories/tutorials/dual-axes.ts:51]:
```suggestion
however it is best to be explicit when working with dual axes charts. If \`main\` is not specified, the library
```

**Review by zvonimirfras — CHANGES_REQUESTED** [review]

File: packages/core/stories/tutorials/dual-axes.ts [hunk 10]
```diff
@@ -0,0 +1,54 @@
+import marked from "marked";
+
+export const dualAxesTutorial = {
+	name: "Dual axes charts",
+	content: marked(`
+# Dual Axes Charts
+
+If a chart is considered to be a **dual axes** chart, it means that the chart uses 2 **different** axes for visually mapping ranges for the data.
```
Comment (theiliad) [inline packages/core/stories/tutorials/dual-axes.ts:8]:
```suggestion
If a chart is considered to be a **dual axes** chart, it means that the chart uses 2 **different** axes for visually mapping [ranges](https://www.mathsisfun.com/sets/domain-range-codomain.html) for the data.
```

File: packages/core/stories/tutorials/combo-charts.ts [hunk 11]
```diff
@@ -49,7 +49,7 @@ const comboOptions = {
 			type: ChartTypes.LINE // use a valid ChartType from enums
 			options: {
 				points: {
-					enabled: false; // disable the point on the LineChart won't affect the points in the AreaChart
+					enabled: false // disable the point on the LineChart won't affect the points in the AreaChart
```
Comment (theiliad) [inline packages/core/stories/tutorials/combo-charts.ts:52]:
```suggestion
					enabled: false // disabling the scatter points on the LineChart here won't affect the points in the AreaChart
```

File: packages/core/stories/tutorials/combo-charts.ts [hunk 12]
```diff
@@ -96,7 +96,7 @@ const comboOptions = {
 			// add the base options for that type of chart if it is in the carbon-charts library and any additional configurations needed
 			options: Object.assign({}, options.lineChart,  {
 				points: {
-					enabled: false; // disable the point on the LineChart won't affect the points in the AreaChart
+					enabled: false // disable the point on the LineChart won't affect the points in the AreaChart
```
Comment (theiliad) [inline packages/core/stories/tutorials/combo-charts.ts:99]:
```suggestion
					enabled: false // disabling the scatter points on the LineChart here won't affect the points in the AreaChart
```

**Review by theiliad — CHANGES_REQUESTED** [review]
g2g otherwise

**Review by cal-smith — DISMISSED** [review]

**Review by theiliad — APPROVED** [review]

**Review by cal-smith — APPROVED** [review]

**Pull request merged**